test/functional/core/startup_spec.lua

```diff
@@ -251,6 +251,23 @@ describe('startup', function()
                                                                   |
     ]])
   end)
+
+  it('sets \'shortmess\' when loading other tabs', function()
+    clear({args={'-p', 'a', 'b', 'c'}})
+    local screen = Screen.new(25, 4)
+    screen:attach()
+    screen:expect({grid=[[
+  {1: a }{2: b  c }{3:               }{2:X}|
+  ^                         |
+  {4:~                        }|
+                           |
+    ]], attr_ids={
```

Comment:
just for future reference: this could be indented, `screen:expect()` will ignore the indent whitespace.